File: PharmaDicBackEnd/PharmaDicBackEnd.ApiService/DTOs/MedicineIngredientInputDto.cs
using System.ComponentModel.DataAnnotations;

namespace PharmaDicBackEnd.ApiService.DTOs
{
    public class MedicineIngredientInputDto
    {
        [Required(ErrorMessage = "ID Hoạt chất không được để trống.")]
        public int IngredientId { get; set; }
        public string? Amount { get; set; }
    }
}

File: PharmaDicMobile/app/src/main/java/com/example/pharmadicmobile/dtos/MedicineIngredientInputDto.java
package com.example.pharmadicmobile.dtos;

public class MedicineIngredientInputDto {
    private int ingredientId;
    private String amount;

    public MedicineIngredientInputDto() {}

    public int getIngredientId() { return ingredientId; }
    public void setIngredientId(int ingredientId) { this.ingredientId = ingredientId; }

    public String getAmount() { return amount; }
    public void setAmount(String amount) { this.amount = amount; }
}
